wai-websockets/Network/Wai/Handler/WebSockets.hs

```diff
@@ -49,7 +49,7 @@ isWebSocketsReq req =
 --         'WS.sendTextData' conn ("Hello, client!" :: 'Data.Text.Text')
 --
 --     backupApp :: 'Wai.Application'
---     backupApp = 'Wai.respondLBS' 'Network.HTTP.Types.status400' [] "Not a WebSocket request"
+--     backupApp _ response = response $ 'Wai.responseLBS' 'Network.HTTP.Types.status400' [] "Not a WebSocket request"
```

Comment:
Typical nomenclature here would be `respond` instead of `response`, you OK with that change?